packages/visual-styling/lib/colors.js

```diff
@@ -22,12 +22,7 @@
  * SOFTWARE.
  */
 
-export interface ColorsArray extends Array<string> {
-    primary: string[];
-    full: string[];
-}
-
-const colors: ColorsArray = <any>[
+var colors = [
```

Comment:
Yes, instead of having a `src` directory and a `lib` directory.  I decided to just have a `lib` directory since it is just this single file, and having a copy command/a compiling step is unnecessary.